lewis/core/control_server.py

```diff
@@ -191,6 +194,22 @@ def add_object(self, obj, name):
             glue = '.' if not method_name.startswith(':') else ''
             self._add_function(name + glue + method_name, exposed_object[method_name])
 
+    def remove_object(self, name):
```

Comment:
As I wrote in the PR notes above, we're going to need it for other tickets when we start switching out objects in the simulation.